playbooks/roles/openshift_install/templates/install-config.yaml.j2

```diff
@@ -21,6 +21,7 @@ networking:
   - 172.30.0.0/16
 platform:
   none: {}
+fips: {{ install_fips_mode }}
```

Comment:
You will need to add a default for this in the `openshift_install` role so that it will work if someone doesn't define it.